### DISCUSSION TITLE
alpha blending

### DISCUSSION THREAD

**Discussion by jjdubois22** [StjerneIdioten/I3D-Blender-Addon#131] [Help] [answered]
Thank you for a great tool. Is there a way when exporting to GE to have the alpha blending box checked automatically?

**Comment by ssnd292** [comment, marked as answer]
![2021-05-09 18_03_09-support-blender-exporter - Discord](https://user-images.githubusercontent.com/3914410/117578875-d4936080-b0f0-11eb-99c3-701365cdddf0.jpg)

**Reply by jjdubois22** [reply]
very easy, Thank you for the quick response.
